MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -2633,234 +2633,187 @@ public GetBuildInfoResponse GetBuildInfoMultiple (WebServiceLogin login, string
 
 				lanes = db.GetAllLanes ();
 
-				switch (response.Host.QueueManagement) {
-				case DBQueueManagement.OneRevisionWorkAtATime:
```

Comment:
Please still add it as a new option, there may be implementation bugs or performance issues (which typically occurs on Sunday night when most bots have finished working and keep hammering the server every minute for new work, meaning I'll have to fix it on Monday morning when I wake up and find Wrench broken).